src/puppetlabs/puppetdb/config.clj

```diff
@@ -316,13 +317,20 @@
   (let [default-web-router-service
         {:puppetlabs.puppetdb.cli.services/puppetdb-service "/pdb/query"
          :puppetlabs.puppetdb.dashboard/dashboard-service "/pdb"
-         :puppetlabs.puppetdb.metrics/metrics-service "/metrics"}]
+         :puppetlabs.puppetdb.metrics/metrics-service "/metrics"}
+        bootstrap-cfg (-> (find-bootstrap-config config-data)
+                          slurp
+                          str/split-lines)
+        dashboard-redirect? (contains? (set bootstrap-cfg)
+                                       "puppetlabs.puppetdb.dashboard/dashboard-redirect-service")]
     (doseq [[svc route] default-web-router-service]
       (when (get-in config-data [:web-router-service svc])
         (-> (format "Configuration of the `%s` route is not allowed. This setting defaults to `%s`." svc route)
             utils/println-err)))
     ;; We override the users settings as to make the above routes *not* configurable
-    (update config-data :web-router-service merge default-web-router-service)))
+    (update config-data :web-router-service merge default-web-router-service
+            (when dashboard-redirect?
+              {:puppetlabs.puppetdb.dashboard/dashboard-redirect-service "/"}))))
```

Comment:
I thought this had to be `""` not `"/"`, does `"/"` not serve the redirect at `//`?